src/Util/AdminObjectAclData.php

```diff
@@ -76,22 +74,18 @@ class AdminObjectAclData
 
     /**
      * @var string
-     *
```

Comment:
We generally add a line between different tag category, like  `@foo` et `@phpstan-foo`